script/1_0_2current_en_script/Arcueid/Day 7/07_00_ARC07_1B.txt

```diff
@@ -1,82 +1,82 @@
 <Page4>
-However, I wasn't able to join any club activities.
-Because I have chronic anemia, I'm afraid that I'll be a nuisance to the others when the time comes. And besides, my doctor also strongly insists that I refrain from anything that involves excessive movement.
+However, I wasn't able to join any clubs.
+Because I have chronic anemia, I'm afraid that I'll be a nuisance to the others at an important time. And besides, my doctor also strongly insists that I refrain from anything that involves excessive movement.
 <Page5>
 I was also invited to club activities back when I was a freshman.
 Whenever that happened, I always turned them down by saying 'maybe next time'.
 Each time that I did that, I felt an indescribable sense of distance between us.
 <Page6>
 ...In the end, that's what happened.
-I'm sure it was part of my self-defense mechanism telling me unconciously to not get involved with them under any circumstance.
+I'm sure it was part of my self-defense mechanism unconsciously telling me not to get involved with them under any circumstances.
 <Page7>
 "..."
 I put an end to my boring sentiments.
-I send my support to the hard-working club members from the bottom of my heart and walk toward the school building.
+I cheer on the hard-working club members from the bottom of my heart and walk toward the school building.
 <Page8>
 "O-Oh my. Good morning Shiki-kun, aren't you a bit early this morning?"
 "...Yes, I got the time a bit mixed up.
 Good morning, Noel-sensei."
 <Page9>
 On my way to the classroom, I come across Noel-sensei unexpectedly.
-Just like me, Noel-sensei didn't expect anyone to be here at such an hour and looks a little dumb-founded.
+Just like me, it seems Noel-sensei didn't expect anyone to be here at such an hour and looks a little dumb-founded.
 <Page10>
-"Rather than getting the time mixed up and coming to school late, you came to school early instead. You must be one of those types, the hard working ones right? But while you're still young, wouldn't it be better to live a bit more relaxed?"
-"But I'm more than relaxed. Besides, I don't even like to study.
-More importantly, I want to talk about my truancy from last Friday..."
+"Rather than getting the time mixed up and coming to school late, you came to school early instead. You must be one of those types, the hard working ones right? But while you're still young, wouldn't it be better to loosen up a little bit?"
+"I'm plenty laid-back. Besides, I don't even like to study.
+More importantly, I want to talk about my unnotified absence from last Friday..."
 <Page11>
-"Oh, you mean that. Don't worry, I didn't contact your home.
-It's totally normal to boycott every once in a while. It would have been more convenient for you this way, right Shiki-kun?"
+"Oh, you mean that. Don't worry, I didn't contact your family.
+It's totally normal to boycott every once in a while. Besides, it's more convenient for you this way, right Shiki-kun?"
```

Comment:
A boycott every once in a while is totally normal.

Also please put a comment that its literally ボイコット because my first thought was "what a weird word choice"